### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Sitewide Facilities: Facility Locator Services Autosuggest + Progressive Disclosure

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#101887]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** If you decide to build your own combobox component separate from what's already underway in the design system, make sure it has all of the accessibility functionality it needs to be functional for expected keyboard actions and have everything announced to screen reader users. The work done by the 526ez team is a great place to start. I'll also point to this [blog post from Pope Tech](https://blog.pope.tech/2024/07/01/create-an-accessible-combobox-using-aria/), especially the sections on required keyboard actions and required ARIA roles, states, and attributes.
- [x] **Consider:** @humancompanion-usds raised an interesting question at the end of the meeting about the [width of the inputs on a tablet view](https://www.figma.com/design/GvNVeG9GAXZaBpny7vNxZP/Facility-Locator?node-id=963-9555&t=lWVNWBUlL2rRb8jn-4) and the potential impact on screen magnification / pinch-and-zoom users, eg. an unexpected interaction for something that at a certain magnification just looks like a text input. After some discussion with other accessibility folks, the standard we've landed on is: the maximum width of an input should be relatively close to the maximum width of an expected value for the input. In other words, whatever the longest service type we support is, the service type input shouldn't be much wider than what it takes to display that text.
   
   This is not a super concrete standard! It's also not something that's going to keep me up at night, since it shouldn't prevent anyone from completing a task. But I'd encourage you to explore hashing out that kind of width limit and contributing it back to the design system!

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by Shamiaalam** [comment]
I run through test for A11y which includes screen reader, keyboard interaction or navigation, 526ez team’s work and Pope Tech’s blog for guidance on ARIA roles and attributes. All are working good as an a11y perspective.
